bindings/nodejs/examples/how-tos/simple_transaction/request_funds.ts

```diff
@@ -20,12 +20,14 @@ async function run() {
 
         const account = await wallet.getAccount('Alice');
 
-        const accountAddresses = await account.addresses();
-        console.log('Account addresses:', accountAddresses);
+        const address = (await account.addresses())[2].address;
```

Comment:
```suggestion
        const address = (await account.addresses())[0].address;
```

Reply:
Also CI complains about format

Reply:
A sorry. That was just a small test from my side 😅